cocos/renderer/CCTextureCache.h

```diff
@@ -139,9 +141,10 @@ class CC_DLL TextureCache : public Ref
     * If the image was not previously loaded, it will create a new Texture2D object and it will return it.
     * Otherwise it will return a reference of a previously loaded image.
     * @param key The "key" parameter will be used as the "key" for the cache.
+    * @param generateMipMap need generate mipmap levels(if image does not contain).
     * If "key" is nil, then a new texture will be created each time.
     */
-    Texture2D* addImage(Image *image, const std::string &key);
+    Texture2D* addImage(Image *image, const std::string &key, bool generateMipMap = false);
```

Comment:
we should not use default arguments in instance methods:
https://github.com/cocos2d/cocos2d-x/wiki/Cpp--Coding-Style#default-arguments